scanner/e2etests/helmchart/templates/db-service.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.app.db.name }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  ports:
+  - name: {{ .Values.app.db.name }}
```

Comment:
consider prepending with `tcp`. I think it has to do with Istio, and even though we don't currently use it, I think it's better to rename it now in case something changes in the future

Reply:
73751815e8